icu4c/APIChangeReport.html

```diff
@@ -38,1071 +38,959 @@ <h1>ICU4C API Comparison: ICU 72 with ICU 73</h1>
+<td class="file">displayoptions.h</td><td class="proto">UDisplayOptionsDisplayLength icu::DisplayOptions::getDisplayLength() const</td><td class="" colspan="2" align="center">Draft&rarr;Stable<br>ICU 72</td>
+</tr>
+<tr class="row0">
+<td class="file">displayoptions.h</td><td class="proto">UDisplayOptionsGrammaticalCase icu::DisplayOptions::getGrammaticalCase() const</td><td class="" colspan="2" align="center">Draft&rarr;Stable<br>ICU 72</td>
+</tr>
+<tr class="row1">
+<td class="file">displayoptions.h</td><td class="proto">UDisplayOptionsNameStyle icu::DisplayOptions::getNameStyle() const</td><td class="" colspan="2" align="center">Draft&rarr;Stable<br>ICU 72</td>
+</tr>
+<tr class="row0">
+<td class="file">displayoptions.h</td><td class="proto">UDisplayOptionsNounClass icu::DisplayOptions::getNounClass() const</td><td class="" colspan="2" align="center">Draft&rarr;Stable<br>ICU 72</td>
+</tr>
+<tr class="row1">
+<td class="file">displayoptions.h</td><td class="proto">UDisplayOptionsPluralCategory icu::DisplayOptions::getPluralCategory() const</td><td class="" colspan="2" align="center">Draft&rarr;Stable<br>ICU 72</td>
+</tr>
+<tr class="row0">
+<td class="file">displayoptions.h</td><td class="proto">UDisplayOptionsSubstituteHandling icu::DisplayOptions::getSubstituteHandling() const</td><td class="" colspan="2" align="center">Draft&rarr;Stable<br>ICU 72</td>
+</tr>
+<tr class="row1">
+<td class="file">displayoptions.h</td><td class="proto">icu::DisplayOptions::DisplayOptions(const DisplayOptions&amp;)=default</td><td class="" colspan="2" align="center">Draft&rarr;Stable<br>ICU 72</td>
+</tr>
+<tr class="row0">
+<td class="file">displayoptions.h</td><td class="proto"><tt>static</tt> Builder icu::DisplayOptions::builder()</td><td class="" colspan="2" align="center">Draft&rarr;Stable<br>ICU 72</td>
+</tr>
+<tr class="row1">
+<td class="file">formattednumber.h</td><td class="proto">UDisplayOptionsNounClass icu::number::FormattedNumber::getNounClass(UErrorCode&amp;) const</td><td class="" colspan="2" align="center">Draft&rarr;Stable<br>ICU 72</td>
+</tr>
+<tr class="row0">
+<td class="file">measunit.h</td><td class="proto"><tt>static</tt> MeasureUnit icu::MeasureUnit::getQuarter()</td><td class="" colspan="2" align="center">Draft&rarr;Stable<br>ICU 72</td>
+</tr>
+<tr class="row1">
+<td class="file">measunit.h</td><td class="proto"><tt>static</tt> MeasureUnit icu::MeasureUnit::getTonne()</td><td class="" colspan="2" align="center">Draft&rarr;Stable<br>ICU 72</td>
+</tr>
+<tr class="row0">
+<td class="file">measunit.h</td><td class="proto"><tt>static</tt> MeasureUnit* icu::MeasureUnit::createQuarter(UErrorCode&amp;)</td><td class="" colspan="2" align="center">Draft&rarr;Stable<br>ICU 72</td>
+</tr>
+<tr class="row1">
+<td class="file">measunit.h</td><td class="proto"><tt>static</tt> MeasureUnit* icu::MeasureUnit::createTonne(UErrorCode&amp;)</td><td class="" colspan="2" align="center">Draft&rarr;Stable<br>ICU 72</td>
+</tr>
+<tr class="row0">
```

Comment:
I think the `MeasureUnit` additions above aren't promotions for ICU 72 (I don't remember seeing them when I did the promotions); I think they were "born stable" in ICU 72.

Reply:
The [ICU 72 API change report](https://htmlpreview.github.io/?https://github.com/unicode-org/icu/blob/release-72-1/icu4c/APIChangeReport.html) says that they were added as draft.

Reply:
They got promoted in July in PR #2530.
This may bog down your browser: https://github.com/unicode-org/icu/pull/2530/files?file-filters%5B%5D=.h&show-viewed-files=true

Reply:
@markusicu, thanks for figuring this out.  It did seem weird that I didn't see this when I did the promotions.  But if they just got promoted in July, I guess they _do_ belong in this report and I was wrong to flag that.

@DraganBesevic and @pedberg-icu , *should* the units have gotten promoted to stable as part of that integration?  Is that how the process normally works for measurement units?  Or was there a hiccup this time?  I thought I remembered doing measurement units as part of the API promotions in the past.